### ISSUE TITLE
[FEATURE] Refactor kube waitForPhase

### ISSUE THREAD

**Issue by eddie-knight** [citihub/probr#323]
**Is your feature request related to a problem? Please describe.**
no

**Describe the solution you'd like**
The function `service_packs\kubernetes\kube.go#waitForPhase` should be reviewed for efficiency, and at minimum the log "...which we're not waiting on" should be revised or use a lower priority loglevel.

**Describe alternatives you've considered**
n/a

**Additional context**
n/a
